ci-operator/step-registry/openshift/e2e/test/qe/upgrade/openshift-e2e-test-qe-upgrade-chain.yaml

```diff
@@ -0,0 +1,11 @@
+chain:
+  as: openshift-e2e-test-qe-upgrade
+  steps:
+  - ref: cucushift-upgrade-setchannel
+  - ref: cucushift-upgrade-check-not-recommended
+  - ref: cucushift-upgrade-prehealthcheck
+  - ref: cucushift-upgrade-toimage
+  - ref: cucushift-upgrade-healthcheck
+  - chain: openshift-e2e-test-qe
+  documentation: |-
+    Upgrade the cluster then execute e2e tests from QE, which include golang (openshift-extended-test), cucushift (cucushift-e2e), cypress (openshift-extended-web-tests), ...(more to add)
```

Comment:
The chain executes `upgrade`, then `e2e`.
So instead of `openshift-e2e-test-qe-upgrade`, perhaps use `openshift-upgrade-qe-test-e2e` ?

Reply:
Chain moved/renamed.